README.md

```diff
@@ -202,11 +203,37 @@ model_repository/
 
 ```
 
+### Reshaping model to support dynamic batch size or dimension
 
-## Known Issues
+Following section shows how to use OpenVINO dynamic shapes. `-1` denotes dimension accepting any value on input. In this case
+while model originally accepted input with layout `NCHW` and shape `(1,3,224,224)`, now it accepts any batch size and height.
+
+*Note*: If model is originally exported with dynamic shapes support there is no need to manually specify dynamic shapes in config.
 
-* Models with dynamic shape are not supported in this backend now.
+```
+input [
+  {
+    name: "input"
+    data_type: TYPE_FP32
+    dims: [ -1, 3, -1, 224]
```

Comment:
```suggestion
    dims: [ -1, 3, -1, -1]
```

Reply:
This is now inconsistent with comment in L209